pyproject.toml

```diff
@@ -35,17 +34,18 @@ classifiers = [
 poetry_installer = "tox_poetry_installer"
 
 [tool.poetry.extras]
-poetry = ["poetry"]
+poetry = ["poetry", "cleo"]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
-poetry = {version = "^1.0.0", optional = true}
-poetry-core = "^1.0.0"
+python = "^3.7"
+cleo = {version = "^1.0.0a5", optional = true, allow-prereleases = true}
+poetry = {version = "1.2.0b3", optional = true, allow-prereleases = true}
+poetry-core = {version = "1.1.0b3", allow-prereleases = true}
```

Comment:
Can you bump these to the stable versions?

Reply:
Cleo is still alpha-5 even with poetry 1.2... but the rest :+1: